test/VocabularyTest.cpp

```diff
@@ -116,17 +116,26 @@ TEST(VocabularyTest, IncompleteLiterals) {
 }
 
 TEST(Vocabulary, PrefixFilter) {
-  RdfsVocabulary voc;
-  voc.setLocale("en", "US", true);
-  ad_utility::HashSet<string> s;
+  RdfsVocabulary vocab;
+  vocab.setLocale("en", "US", true);
+  ad_utility::HashSet<string> words;
```

Comment:
```suggestion
  ad_utility::HashSet<string> words{{"firstWord", "secondWord"}};
```
(a little bit more concise than the explicit `insert` calls).